### DISCUSSION TITLE
Readme.md deaths

### DISCUSSION THREAD

**Discussion by TheRealSvc** [owid/covid-19-data#2600] [Ideas]
From reading the readme it appears there could be only a underreporting of deaths due to limited testing capabilities. 
But certainly overreporting is possible if the distinction between "died due to corona" or "died with a positive test result but not caused by corona". Since Omicron is rather mild and at the same time very contagious, there are now many cases where C19 is only to be seen as a side diagnosis. E.g. Some german hospitals report that only 20% of ICU Covid patients were admitted to hospital due to C19. The other 80% are ICU patients who had to be moved to the C19 isolation station because of a positive Test result. Nevertheless (at least in germany) they would be counted as C19 deaths. There is also a big problem of consistency. 
Proposal: Update the Readme part about the quality of reported deaths and mention the possibility of overreporting.

**Comment by edomt** [comment]
@TheRealSvc Can I ask specifically what part of the README you're referring to? I don't think there's any mention of limited testing capabilities on https://github.com/owid/covid-19-data/tree/master/public/data so I'm probably looking at the wrong document.

@lucasrodes If needed, we can use the new sentence that we've started using on our charts recently, which better encompasses both underreporting and overreporting: 

> Due to varying protocols and challenges in the attribution of the cause of death, the number of confirmed deaths may not
accurately represent the true number of deaths caused by COVID-19.

**Reply by lucasrodes** [reply]
Added support for notes to our README in 163bf8ad67d34ea0a7dfa65fd81ab023ce984863.

We now have[ notes on confirmed deaths.](https://github.com/owid/covid-19-data/tree/master/public/data#notes)

**Reply by TheRealSvc** [reply]
It seems i was wrong about the Readme.md. Currently i've only limited access only via mobile phone.

I could only find a similar phrase in the production bundle owid/ovid-static/coronavirus/country/germany.html (and elsewhere)

Where in a title it says "likely to be higher".
(If higher or lower clearly depends on the country)

Sorry, i should'nt do this from a phone i guess. Once "back to normal" i will check if i can find it.

**Comment by TheRealSvc** [comment]
For instance on the Landing page for deaths https://ourworldindata.org/covid-deaths
you still write
"We know – based on reports and estimates of excess deaths – that these figures underestimate the total impact of the pandemic on mortality globally."
If you still think this is true, you should at least provide the sources that lead you to this conclusion. Because this statement goes beyond mere reporting.

**Comment by TheRealSvc** [comment]
picking that up again from here  https://ourworldindata.org/covid-deaths

"This page provides data on the number of confirmed deaths from COVID-19.
Research has shown that these figures are an underestimate of the total pandemic death toll. This is because of limited testing, poorly functioning death registries, challenges in determining the cause of death, and disruptions during the pandemic"

There is no evidence that there is more undereporting of deaths due to limited capabilities, then there is overcounting by counting all deaths with positive PCR test as a covid death, despite the root cause was different.  

This sentence is misinformation. You should delete it!
